paparazzi/paparazzi-gradle-plugin/build.gradle

```diff
@@ -50,6 +52,10 @@ tasks.named('compileKotlin').configure {
   it.dependsOn(generateVersion)
 }
 
+tasks.withType(SpotlessTask).configureEach {
+  it.dependsOn(generateVersion)
+}
+
```

Comment:
hmm, why does a layoutlib upgrade require updates to spotless tasks?

let's move any "on-the-fly" changes instead to separate smaller PRs.

Reply:
ah, https://github.com/cashapp/paparazzi/actions/runs/4758346202/jobs/8456286196

...grumble grumble...

Reply:
I think that's a different issue coming from Gradle 8.x upgrade. Based on that error message my guess is that there are some tests which "generate" Kotlin files and Spotless eagerly picks them up even though they're outside of project's sourceSets.

Reply:
the error is correct, it's just an error in Gradle 8.x now